src/net.ts

```diff
@@ -13,14 +13,24 @@ export default class Network {
    */
   constructor (provider: Provider) {
     this.provider = provider;
-    if (!VERSION_LIST[SDK_VERSION]) {
-      throw Error("Current sdk version doesn't exist in the list");
+    this.protoVer = '0.3.1';
```

Comment:
Could this be defined as a constant somewhere?

Reply:
Done!